app/utils/EmployeeTypeUtil.scala

```diff
@@ -36,34 +36,34 @@ trait EmployeeTypeUtil extends FeatureSwitching {
     type2bEmployeeResult: Option[T] = None)(implicit request: DataRequest[_], appConfig: FrontendAppConfig): Option[T] =
     (request.userAnswers.getV(RegularLengthEmployedPage), request.userAnswers.getV(ClaimPeriodStartPage)) match {
       case (Valid(RegularLengthEmployed.Yes), _) =>
-        logger.debug("[EmployeeTypeUtil][regularPayResolver] Type 1 Employee")
+        Logger.debug("[EmployeeTypeUtil][regularPayResolver] Type 1 Employee")
```

Comment:
Isn't this logger deprecated? Feels like the one which was imported before was correct?

Reply:
They are exactly the same under the hood. There was no real reason to do this I just think it looks better

Reply:
Why this gets bumped to Play27 though using the apply on the `Logger` object is marked as a deprecated - assuming support will be removed in Play28 for this. So it's safer to keep it as it was with `play.api.Logger.logger` and a lower case `logger` in the code. That's what we've had to change the other services to.